addons/binding/org.openhab.binding.tellstick/ESH-INF/thing/devices.xml

```diff
@@ -72,14 +72,14 @@
 				<description>The name of the device</description>
 				<required>false</required>
 			</parameter>
-			<parameter name="deviceId" type="integer">
+			<parameter name="deviceId" type="text">
 				<label>DeviceID</label>
 				<description>This is the unique id of a device.</description>
 				<required>true</required>
 			</parameter>
 
-			<parameter name="resend" type="integer">
-				<label>Resend</label>
+			<parameter name="repeat" type="integer">
```

Comment:
@jarlebh @kaikreuzer is this a backwards compatible change, if not should it be mentioned?

https://github.com/openhab/openhab-distro/wiki/openHAB-2.1-Release-Notes